sympy/integrals/intpoly.py

```diff
@@ -128,8 +138,7 @@ def polytope_integrate(poly, expr=None, *, clockwise=False, max_degree=None):
         return result
 
     if expr is None:
-        raise TypeError('Input expression be must'
```

Comment:
this could be fixed by putting a space before the closing single quote

Reply:
Yeah I could do that but still the message text would need to be changed since `" Input expression be must be a valid SymPy expression "` doesn't sound right . Is the current message in my diff not correct ?

Reply:
I think the error message is appropriate .I have made it uniform throughout `intpoly.py` now